extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -341,26 +338,6 @@ public function onShowDiff( \DifferenceEngine $diff, &$notice ) {
 		return true;
 	}
 
-	public function onUserRightsGroupCheckboxes( $group, &$disabled, &$irreversible ) {
-		global $wgUser;
-
-		if ( $group === 'content-reviewer' && ( !$wgUser->isAllowed( 'content-review' ) || !$wgUser->isStaff() ) ) {
-			$disabled = true;
-		}
-
-		return true;
-	}
-
-	public function onUserAddGroup( \User $user, $group ) {
-		global $wgUser;
-
-		if ( $group === 'content-reviewer' && ( !$wgUser->isAllowed( 'content-review' ) || !$wgUser->isStaff() ) ) {
```

Comment:
onUserAddGroup is part of a check inside of the user permissions service. As for GroupCheckboxes I moved that logic directly to the special page displaying it